.azure-pipelines/client.test.live.yml

```diff
@@ -9,36 +9,46 @@ jobs:
           pomFilePath: './keyvault/data-plane/pom.xml'
         AzureEventHubs:
           pomFilePath: './eventhubs/data-plane/pom.xml'
+        AzureStorage:
+          pomFilePath: './storage/data-plane/pom.xml'
 
     pool:
       vmImage: 'vs2017-win2016'
 
     steps:
-      # Adding -Dmaven.wagon.http.pool=false to the mvn command as a workaround to prevent build failures at maven artifacts downloading stage.
-      - task: Maven@3
-        displayName: 'Run Live tests'
-        inputs:
-          mavenPomFile: $(pomFilePath)
-          options: '--batch-mode -Dmaven.wagon.http.pool=false -Dsurefire.rerunFailingTestsCount=3'
-          mavenOptions: '-Xmx3072m -Dorg.slf4j.simpleLogger.defaultLogLevel=error -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn'
-          javaHomeOption: 'JDKVersion'
-          jdkVersionOption: '1.8'
-          jdkArchitectureOption: 'x64'
-          publishJUnitResults: false
-          goals: 'test'
-        env:
-          ARM_CLIENTID: $(java-keyvault-test-arm-client-id)
-          ARM_CLIENTKEY: $(java-keyvault-test-arm-client-key)
-          AZURE_TEST_MODE: RECORD
-          AZURE_BATCH_ACCOUNT: $(java-batch-test-account-name)
-          AZURE_BATCH_ACCESS_KEY: $(java-batch-test-account-access-key)
-          STORAGE_ACCOUNT_KEY: $(java-batch-test-storage-account-key)
-          STORAGE_ACCOUNT_NAME: $(java-batch-test-storage-account-name)
-          AZURE_BATCH_ENDPOINT: https://servbatch.centralus.batch.azure.com
-          AZURE_EVENTHUBS_CONNECTION_STRING: ${java-event-hubs-test-connection-string}
+    # Adding -Dmaven.wagon.http.pool=false to the mvn command as a workaround to prevent build failures at maven artifacts downloading stage.
```

Comment:
Looks like the indention was messed up as pat of your change can you please indent this section one more level.

Reply:
Fixed the indentation.